src/api/Users.js

```diff
@@ -36,6 +36,19 @@ class Users extends Base {
         return `${this.getBaseApiUrl()}/users/${id}/avatar`;
     }
 
+    /**
+     * API URL for fetching all users in the current user's enterprise
+     *
+     * @param {string} filterTerm Filter for enterprise users
+     * @return {string} URL for fetching enterprise users
+     */
+    getUsersInEnterpriseUrl(filterTerm: string): string {
+        if (!filterTerm) {
```

Comment:
Hm, after looking at this, looks like we treat `''` as a possible filter term by default on LN123? It that's the case, when using the default value, it will throw?

Reply:
@booc0mtaco Thanks for catching this. I updated the PR based on your comments -- could you take another look?